README.md

```diff
@@ -31,7 +31,7 @@ defmodule AudioGenerating.Pipeline do
   def handle_init(_) do
```

Comment:
There's more to update here
```suggestion
  def handle_init(_ctx, _opts) do
```
plus links, returned values etc. Ensure this can be run